R/derive_param_confirmed_bor.R

```diff
@@ -381,11 +381,18 @@ derive_param_confirmed_bor <- function(dataset,
     )
   }
 
+  # Check for CR followed by PR (this should not occur in clean data)
+  signal_crpr(
+    source_data,
+    order = vars(ADT)
+  )
+
```

Comment:
I would not call `signal_crpr()` before `filter_pd()` because it may confuse the user if a warning is issued although the PR after PD is removed by `filter_pd()`.